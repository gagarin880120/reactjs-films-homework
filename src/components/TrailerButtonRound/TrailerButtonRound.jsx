import React from 'react';
import styles from './TrailerButtonRound.module.scss';

export default function TrailerButtonRound(props) {
  return (
    <button
      testid="trailerButtonRound"
      onClick={() => {
        props.onTrailerButtonClick(props.id);
      }}
      className={styles.trailerButton}
    >
      <i className="fas fa-play"></i>
    </button>
  )
}
