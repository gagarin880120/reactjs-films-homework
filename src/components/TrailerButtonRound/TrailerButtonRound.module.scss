.trailerButton {
  background: #3bb3df;
  color: #fff;
  border-radius: 50%;
  border: 0;
  width: 65px;
  height: 65px;
  font-size: 40px;
  outline: 0;
  line-height: 65px;
  cursor: pointer;
}

.trailerButton:active {
  transform: translateY(2px);
}

i {
  margin-left: 7px;
}
