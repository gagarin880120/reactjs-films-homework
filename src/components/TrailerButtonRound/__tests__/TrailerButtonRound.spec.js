import ShallowRenderer from 'react-test-renderer/shallow';
import React from 'react';
import TrailerButtonRound from '../TrailerButtonRound';

test('TrailerButtonRound component should render without crashing', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<TrailerButtonRound />);
  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});
