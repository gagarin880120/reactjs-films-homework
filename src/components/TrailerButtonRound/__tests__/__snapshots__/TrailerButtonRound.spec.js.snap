// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrailerButtonRound component should render without crashing 1`] = `
<button
  className="trailerButton"
>
  â–¶
</button>
`;
